PluginGenerator/JarBuilder.cs

```diff
@@ -142,7 +142,7 @@ public bool Build(string fullJarPath)
                 throw new InvalidOperationException(UIResources.JarB_JDK_NotInstalled);
             }
 
-            string tempPath = Path.Combine(Path.GetTempPath(), ".jarBuilder", Guid.NewGuid().ToString());
+            string tempPath = Path.Combine(Utilities.CreateTempDirectory(JarBuilderRootFolderName), Guid.NewGuid().ToString());
```

Comment:
Change the jar builder so that it writes it's temporary files under ".sonarqube.sdk" to be consistent with the other classes.